lib/reporters/xunit.js

```diff
@@ -130,7 +130,7 @@ XUnit.prototype.test = function(test) {
 
   if (test.state === 'failed') {
     var err = test.err;
-    this.write(tag('testcase', attrs, false, tag('failure', {}, false, cdata(escape(err.message) + '\n' + err.stack))));
```

Comment:
Oh, okay, if they are escaped, it shouldn't be necessary. I wonder why it was used in the first place.